src/app/Fake.Core.Target/Target.fs

```diff
@@ -577,6 +577,11 @@ module Target =
     let activateBuildFailure name =
         let t = get name // test if target is defined
         getBuildFailureTargets().[name] <- true
+        
+    /// Deactivates the build failure target.
+    let deactivateBuildFailure name =
+        let t = get name // test if target is defined
+        getBuildFailureTargets().[name] <- false
```

Comment:
Because I have a situation whereby I activate it, but then decide (as a result of a step) to deactivate.

This was possible in fake 4 by modifying the target directly.  Since fake 5 doesn't expose the targets, this is the alternative

Reply:
So you actually call this within a build step. I'm not sure if we should allow that at all :/
Basically I consider these APIs only for "setup" changing things at runtime should probably be done by using the context parameter. The point is when using parallel builds these kind of things make it hard to get it correct.

Reply:
Is the context the argument passed to each target function?

I typically just do "fun _ ->” but that _ is context?

Reply:
Yes exactly (new in fake 5). However the feature you are after is not available yet.

Reply:
Oooo ok, let's close this in favour of adding into context as your right, that makes more sense 

My use case us to halt a final target when failure fires.

Maybe a "success target" would also work for me...

Reply:
Please feel free to change this PR accordingly. In the mean time we can leave it open as "feature-request" or maybe just merge it (will have to think about it and look a bit in the code)